package-lock.json

```diff
@@ -3979,6 +3979,14 @@
       "integrity": "sha1-w2GT3Tzhwu7SrbfIAtu8d6gbHA8=",
       "dev": true
     },
+    "monotone-convex-hull-2d": {
```

Comment:
Ah yes, it's because #400 forgot to modify it. #413 fixes it in its last commit, but this one will pick this diff until #413 is merged. Anyway, this is auto-generated, not sure if it is worthy to review it TBH.